src/MizarData/include/MizarData/ActiveFunctionTimePerFrameComparator.h

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef MIZAR_STATISTICS_ACTIVE_FUNCTION_TIME_PER_FRAME_COMPARATOR_H_
+#define MIZAR_STATISTICS_ACTIVE_FUNCTION_TIME_PER_FRAME_COMPARATOR_H_
+
+#include <utility>
+
+#include "ClientData/ScopeStats.h"
+#include "MizarData/BaselineOrComparison.h"
+#include "MizarData/SamplingWithFrameTrackComparisonReport.h"
+#include "Statistics/Gaussian.h"
+#include "Statistics/StatisticsUtils.h"
+
+namespace orbit_mizar_data {
+
+// TODO(b/236130122) it should be moved to a new library
```

Comment:
Why?

Reply:
Because it's not a data class, it doesn't read data, it doesn't do smth straightforward with it. It implements a statistical methodology and IMO it should belong to MizarStatistics (not to Statistics, as the latter is rather for general-purpose statistics, yet the method is highly Mizar-specific).